qiskit_ionq/helpers.py

```diff
@@ -344,7 +343,7 @@ def decompress_metadata_string_to_dict(input_string):  # pylint: disable=invalid
     return json.loads(decompressed)
 
 
-def qiskit_to_ionq(circuit, backend, passed_args=None):
+def qiskit_to_ionq(circuit, backend, passed_args=None, _params=None):
```

Comment:
_params is a bit awkwardly named, imo, it makes me think QueryParams, and the underscore seems to indicate you shouldn't actually use it (_ stuff in python is usually 'private')

Can we name it like, extra_request_payload or something like that?

Sorry to nitpick so much on this bit